README.md

```diff
@@ -1,21 +1,480 @@
+not versioned, the series' shortname is identical to the spec's shortname.
+
+The `shortname` property is always set.
+
+
+#### `series.currentSpecification`
+
+The shortname of the spec that should be regarded as the current level or
+version in the series. The current spec in a series is up to the group who
+develops the series. In most cases, the current spec is the latest level or
+version in the series that is a "full" spec (see
+[`seriesComposition`](#seriescomposition)).
+
+The `currentSpecification` property is always set.
+
+
+### `seriesVersion`
+
+The level or version of the spec, represented as an `x`, `x.y` or `x.y.z` string
+with `x`, `y` and `z` numbers, and `x` always greater than or equal to `1`. For
+instance, this property will have the value `1.2` (as a string, so enclosed
+in `"`) for the WAI-ARIA 1.2 spec.
+
+The `seriesVersion` property is only set for specs that have a level or version
+number.
+
+
+### `seriesComposition`
+
+Whether the spec is a standalone spec, or whether it is a delta spec over the
+previous level or version in the series. Possible values are `full` or `delta`.
+
+The `seriesComposition` property is always set.
+
+
+### `seriesPrevious`
+
+The `shortname` of the previous spec in the series.
+
+The `seriesPrevious` property is only set where there is a previous level or
+version.
+
+
+### `seriesNext`
+
+The `shortname` of the next spec in the series.
+
+The `seriesNext` property is only set where there is a next level or version.
+
+
+### `release`
+
+An object that represents the latest published snapshot of the spec, when it
+exists.
+
+The `release` property is only set for W3C specs published as TR documents.
+
+
+#### `release.url`
+
+The URL of the latest published snapshot of the spec. Matches the versioned
+URL (see [`url`](#url)).
+
+The `url` property is always set.
+
+
+### `nightly`
+
+An object that represents the latest Editor's Draft of the spec, or the living
+standard when the concept of Editor's Draft does not exist.
+
+The `nightly` property is always set.
+
+
+#### `nightly.url`
+
+The URL of the latest Editor's Draft or of the living standard.
+
+The URL is either retrieved from the [W3C API](https://w3c.github.io/w3c-api/)
+for W3C specs, or [Specref](https://www.specref.org/). The document at the
+versioned URL is considered to be the latest Editor's Draft if the spec does
+neither exist in the W3C API nor in Specref. The [`source`](#source) property
+details the actual provenance.
+
+The `url` property is always set.
+
+
+### `source`
+
+The provenance for the `title` and `nightly` property values. Can be one of:
+- `w3c`: information retrieved from the [W3C API](https://w3c.github.io/w3c-api/)
+- `specref`: information retrieved from [Specref](https://www.specref.org/)
+- `spec`: information retrieved from the spec itself
+
+The `source` property is always set.
+
+
+## How to add/update/delete a spec
```

Comment:
should this section be moved to CONTRIBUTING.md? (while keeping a reference to it from here)

Reply:
I'm not sure I like the back-and-forth that this creates between the contributing file that explains which properties to include, and the readme file where the properties are defined, but anyway, too late, now done ;)